lib/tasks/oauth_access_records.rake

```diff
@@ -7,20 +7,39 @@ namespace :oauth_access_records do
   desc "Delete expired OAuth access grants and tokens"
   task delete_expired: :environment do
     klasses = [Doorkeeper::AccessGrant, Doorkeeper::AccessToken]
+    event_log_type_by_record_class = {
+      Doorkeeper::AccessGrant => EventLog::ACCESS_GRANTS_DELETED,
+      Doorkeeper::AccessToken => EventLog::ACCESS_TOKENS_DELETED,
+    }
+
     klasses.each do |klass|
       ids = [nil]
 
       count = 0
 
       until ids.empty?
         ids = klass
-          .where("expires_in is not null AND DATE_ADD(created_at, INTERVAL expires_in second) < ?", Time.zone.now)
+          .where("expires_in is not null AND DATE_ADD(created_at, INTERVAL expires_in second) <= ?", Time.zone.now)
           .limit(1000)
           .pluck(:id)
```

Comment:
Minor: Although it's not directly relevant to the changes we want to make in this PR, it might be nice to take the opportunity to change the code to use the more idiomatic `ActiveRecord::Batches#in_batches` here...?